core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerProxy.scala

```diff
@@ -631,23 +630,27 @@ class ContainerProxy(
             ActivationResponse.whiskError(Messages.abnormalRun))
       }
 
+    val logsToBeCollected = collectLogs.logsToBeCollected(job.action)
+    val sendCompletionAfterLogsCollection = logsToBeCollected == true
```

Comment:
instead of defining these two values which are essentially the same (value), how about just defining one value with this name and using it accordingly `splitAckMessagesPendingLogCollection`?

Reply:
Naming is very hard so thanks for the suggestion :). Updated PR